libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -403,6 +404,24 @@ dictionary StaticBackupResponse {
     sequence<string>? backup;
 };
 
+dictionary CreateOfferRequest {
+    u64? amount_msat;
+    string description;
+    u64? absolute_expiry;
+    string? issuer;
+    u64? supported_quantity;
+    string? label;
+    boolean? single_use;
+};
+
+dictionary PayOfferRequest {
+    string offer;
+    u64? amount_msat;
+    u64? quantity;
```

Comment:
After thinking about it, I agree. Implementing quantity doesn't take much (just a few lines of code in fact, since the changes are handled on CLN) though I do agree in more general terms that perhaps it could be handle elsewhere. Leaving it out for now :+1: